mautrix_instagram/portal.py

```diff
@@ -1263,6 +1267,8 @@ async def _convert_instagram_xma_media_share(
                 f"<strong>{escaped_header_text}</strong>"
                 f"{escaped_caption_text[len(escaped_header_text):]}"
             )
+            content["com.beeper.raw_caption_text"] = caption_text[len(header_text) :]
+        content["com.beeper.instagram_item_username"] = media.header_title_text
```

Comment:
are you sure that this is just the username and not any additional text?

Reply:
Yeah, I'll move it into the if statement where we're already assuming that is the username